### ISSUE TITLE
Add ability to take photos without exif data? 

### ISSUE THREAD

**Issue by pokegamer5547** [KillerInk/FreeDcam#160]
I want to take photo without all additional exif data except for orientation. I think having the ability to take photos without exif data is great for privacy people like me.

**Comment by KillerInk** [comment]
i personal dont think its needed. exif dont contain sensible private data till you enable gps inside freedcam.

**Comment by pokegamer5547** [comment]
@KillerInk yes that's true. But data like device name, and date can be sensitive in some cases.
